### ISSUE TITLE
buggy instrument response triggers evalresp warning on stderr

### ISSUE THREAD

**Issue by trichter** [obspy/obspy#1170]
I don't know how evalresp is wrapped and if it would be easy and desirable to catch this warning and transform it into a Python warning:

```
from obspy import read_inventory
inv = read_inventory()
resp = inv.networks[0].stations[0].channels[0].response
resp.instrument_sensitivity.value *= 5
temp = resp.get_evalresp_response(0.1, 1000)

Output:
 WARNING (norm_resp): computed and reported sensitivities differ by more than 5 percent. 
     Execution continuing.
```

**Comment by krischer** [comment]
The only proper way to really catch the warning would be to add log and warning callbacks to evalresp like is done in `libmseed`. It is probably not too much work but somebody would have to do it.

An alternative would be to catch the output and parse that but that is tricky to get right on all platforms.

**Comment by trichter** [comment]
fixed in #2677
